### PR TITLE
LED-16 Password derusification

### PR DESCRIPTION
**Pull request by tom-in-the-shell**

### CODE REVIEW

File: leda/components/Validation/predefinedValidators.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export const PREDEFINED_VALIDATORS: Record<PredefinedValidator, NormalizedValida
   },
   password: {
     validator: validate.password,
-    invalidMessage: 'Пароль должен содержать не менее 8-и символов, латинские строчные, прописные буквы и цифры',
+    invalidMessage: 'Use at least 8 latin lower- and uppercase letters and numbers',
```
Comment (777PolarFox777) [inline leda/components/Validation/predefinedValidators.ts]:
```suggestion
    invalidMessage: 'Use at least 8 latin lowercase and uppercase letters and numbers',
```
Looks better imo

Comment (777PolarFox777) [reply leda/components/Validation/predefinedValidators.ts]: Do you think it's worth adding special characters as well?

**Review by 777PolarFox777 — APPROVED** [review]
Love the title :D

File: leda/components/Validation/predefinedValidators.ts (hunk 1 above)
Comment (tom-in-the-shell) [reply leda/components/Validation/predefinedValidators.ts]: maybe, as a new feature, you can add it to the board
just numbers look lonely

**Pull request merged**